modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/base_controller.rb

```diff
@@ -20,7 +20,9 @@ def show
           poa_code = BGS::PowerOfAttorneyVerifier.new(target_veteran).current_poa_code
           data = poa_code.blank? ? {} : representative(poa_code).merge({ code: poa_code })
           if poa_code.blank?
-            render json: { data: }
+            raise ::ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
```

Comment:
This raise should move inside the `representative(poa_code)` call. We want an empty response 200 when BGS doesn't give us a poa code. We want a 404 for when BGS gives us a code that we can't find an org or rep for in our db.

Reply:
404 for No PoA in System
200 for No Active PoA
I had the above from the discussion thread, but thought System meant BGS. Thanks for clarification.